lxd/instance/drivers/driver_qemu.go

```diff
@@ -38,7 +38,7 @@ import (
 	"google.golang.org/protobuf/proto"
 	"gopkg.in/yaml.v2"
 
-	"github.com/canonical/lxd/client"
+	lxd "github.com/canonical/lxd/client"
```

Comment:
this tends to get auto added by editors and we dont want it thanks

Reply:
My bad. Fixed!